AdminWebsite/AdminWebsite.AcceptanceTests/Steps/SummarySteps.cs

```diff
@@ -140,6 +140,7 @@ public void WhenTheUserRemovesParticipant()
         public void WhenTheUserRemovesInterpreter()
         {
             _browsers[_c.CurrentUser].Click(SummaryPage.RemoveParticipantLink(GetParticipantBy("Interpreter").Firstname));
+            Thread.Sleep(TimeSpan.FromSeconds(1));
```

Comment:
Did you try waiting for the popup to not be visible first?

Reply:
The popup seems to appear without any issue, but clicking on the pop-up's in quick succession causes signout page issue which causes failure. When I tried it with a pause between clicks helped to overcome this issue!